<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="new_structure_lab">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>160</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>280</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>280</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">[ok='false']{
	background-color: rgb(211, 215, 207); 
	border: 3px solid rgb(32, 74, 135);
}
[ok='true']{
	background-color: rgb(211, 215, 207);
	border: 1px solid black;
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="ok" stdset="0">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>130</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>New structure</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateStructure_btn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>510</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Update structure</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>221</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Compound Id</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="compoundId_eb"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="current_structure_lab">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>160</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>280</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>280</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">[ok='false']{
	background-color: rgb(211, 215, 207); 
	border: 3px solid rgb(32, 74, 135);
}
[ok='true']{
	background-color: rgb(211, 215, 207);
	border: 1px solid black;
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="ok" stdset="0">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current structure in compound</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editStructure_btn">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>450</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit structure</string>
    </property>
   </widget>
   <widget class="QLabel" name="original_structure_lab">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>280</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>280</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">[ok='false']{
	background-color: rgb(211, 215, 207); 
	border: 3px solid rgb(32, 74, 135);
}
[ok='true']{
	background-color: rgb(211, 215, 207);
	border: 1px solid black;
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="ok" stdset="0">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Original ChemReg structure</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>221</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Regno </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="regno_eb"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="regnoBackward_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="regnoForward_btn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>450</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string> &gt; </string>
    </property>
   </widget>
   <widget class="QPushButton" name="compoundForward_btn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>450</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string> &gt; </string>
    </property>
   </widget>
   <widget class="QPushButton" name="compoundBackward_btn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>450</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="regnoCopyStructure_btn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>450</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy structure</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_lab">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>89</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Status: </string>
    </property>
   </widget>
   <widget class="QLabel" name="statusMsg_lab">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>90</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>992</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
